init_friendbot.go

```diff
@@ -35,6 +36,7 @@ func initFriendbot(
 		HTTP:       http.DefaultClient,
 		AppName:    "friendbot",
 	}
+	networkClient := horizonnetworkclient.NewNetworkClient(hclient)
```

Comment:
if new network client factory method could take the url instead such as `horizonnetworkclient.NewNetworkClient(horizonURL)` then could ideally encapsulate the reference to `github.com/stellar/go/clients/horizonclient` to be just internally within `friendbot/internal/horizonnetworkclient` and remove from any other friendbot direct references to the horizon package.